lnwallet/channel.go

```diff
@@ -882,6 +888,10 @@ func (lc *LightningChannel) diskHtlcToPayDesc(feeRate chainfee.SatPerKWeight,
 		ourWitnessScript:   ourWitnessScript,
 		theirPkScript:      theirP2WSH,
 		theirWitnessScript: theirWitnessScript,
+		// Note: we do not persist the experimental endorsement
+		// signal to allow ourselves to easily deprecate it in the
+		// future.
+		IncomingEndorsed: false,
```

Comment:
I think this can be documented in the BLIP, in case of implementation failure set `endorsedByte` to 0.